src/main/webapp/app/exercises/quiz/manage/quiz-question-list-edit-existing.component.ts

```diff
@@ -170,16 +171,58 @@ export class QuizQuestionListEditExistingComponent implements OnChanges {
         if (!this.importFile) {
             return;
         }
+
+        const fileName = this.importFile.name;
+        const fileExtension = fileName.split('.').pop()?.toLowerCase();
```

Comment:
```suggestion
        const fileExtension = fileName.split('.').last()?.toLowerCase();
```
Otherwise dots in the file name could break things